### PR TITLE
Change ANN docs from pb to savedmodel

### PR DESCRIPTION
**Pull request by FelixS90**
This PR addresses an upcoming change in the neural network model format supported by Intellesis - going from [Google Proto-buffer](https://developers.google.com/protocol-buffers) to Tensorflow [SavedModel](https://www.tensorflow.org/guide/saved_model).

The main advantage of SavedModel over .pb is that there is no boilerplate code necessary anymore and input + output nodes can be named arbitrarily.

### CODE REVIEW

**Comment by zeissmicroscopy** [comment]
Reviewed internally.
